gdal/frmts/gtiff/cogdriver.cpp

```diff
@@ -848,10 +1036,27 @@ void GDALRegister_COG()
 "     <Value>FORCE_USE_EXISTING</Value>"
 "     <Value>NONE</Value>"
 "   </Option>"
-"  <Option name='TILING_SCHEME' type='string-select' description='"
-        "Which tiling scheme to use' default='CUSTOM'>"
-"    <Value>CUSTOM</Value>"
-"    <Value>GoogleMapsCompatible</Value>"
+"  <Option name='TILING_SCHEME' type='string' description='"
+        "Which tiling scheme to use pre-defined value or custom inline/outline "
+        "JSON definition' default='CUSTOM'>"
+"    <Value>CUSTOM</Value>";
+
+    const auto tmsList = gdal::TileMatrixSet::listPredefinedTileMatrixSets();
+    for( const auto& tmsName: tmsList )
+    {
+        const auto poTM = gdal::TileMatrixSet::parse(tmsName.c_str());
+        if( poTM &&
+            poTM->haveAllLevelsSameTopLeft() &&
+            poTM->haveAllLevelsSameTileSize() &&
+            !poTM->hasVariableMatrixWidth() )
+        {
+            osOptions += "    <Value>";
+            osOptions += tmsName;
+            osOptions += "</Value>";
+        }
+    }
+
+    osOptions +=
```

Comment:
The metadata help text for ALIGNED_LEVELS needs to be updated to remove WebMercator and state specified tiling scheme. See line 1068

Reply:
good catch. Fixed